Backends/CLX-fb/port.lisp

```diff
@@ -104,11 +98,8 @@
 
 
 (defmethod port-force-output ((port clx-fb-port))
-  (maphash-keys
-   (lambda (key)
-     (when (typep key 'clx-fb-mirrored-sheet-mixin)
-       (%mirror-force-output (sheet-mirror key))))
-   (slot-value port 'climi::sheet->mirror))
+  (maphash-values #'%mirror-force-output
+                  (slot-value port 'climi::sheet->mirror))
```

Comment:
Same issue here. Mapping over values (that is mirrors) is a nice improvements, but the filtering should happen here in my opinion.

Reply:
fixed